base/base.jl

```diff
@@ -278,3 +281,5 @@ immutable Nullable{T}
     Nullable(value::T) = new(false, value)
 end
 
+get(x::Nullable) = x.isnull ? throw(NullException()) : x.value
```

Comment:
Yes, it is required for their use in `string.jl` and `gmp.jl`. 
Moving "nullable.jl" broke other dependencies (`repr`, `@printf` are what I saw) which anyway required a split.
We could also use the `Nullable` members directly instead of using these accessors, but I felt this was better.